docs/guides/nrfconnect_examples_configuration.md

```diff
@@ -142,3 +142,55 @@ Read the
 guide in the nRF Connect SDK's Zephyr documentation if you are interested in
 getting more advanced and detailed information about the configuration
 structure.
+
+<hr>
+
+## Configuring Matter in nRF Connect platform
+
+### Mandatory configuration
+
+To use the Matter protocol, you need to enable the `CONFIG_CHIP` Kconfig option. Setting this option enables the Matter protocol stack and other associated Kconfig options, including CONFIG_CHIP_ENABLE_DNSSD_SRP that is required for the discovery of the Matter device using DNS-SD.
+
+After that, make sure to set the `CONFIG_CHIP_PROJECT_CONFIG` Kconfig option and define the path to the configuration file that specifies Vendor ID, Product ID, and other project-specific Matter settings.
+
+<hr>
+
+### Optional configuration
+
+After enabling the Matter protocol and defining the path to the Matter configuration file, you can enable additional options in Kconfig.
+
+**Sleepy End Device support**
+
+You can enable the support for Thread Sleepy End Device in Matter by setting the following Kconfig options:
+- `CONFIG_OPENTHREAD_MTD`
+- `CONFIG_CHIP_ENABLE_SLEEPY_END_DEVICE_SUPPORT`
+
+**Commissioning with NFC support**
+
+You can configure the Matter protocol to use NFC tag for commissioning, instead of the default QR code.
```

Comment:
```suggestion
You can configure the Matter protocol to use an NFC tag for commissioning, instead of the default QR code.
```
Is there a default QR code, or is a QR code used for commissioning by default?

If the latter, the sentence end should be: ``instead of using a QR code, which is the default configuration.``

Reply:
Again the latter, fixed